clients/java-logger/grpc-test-app/src/main/java/abcxyz/lumberjack/test/talker/TalkerService.java

```diff
@@ -134,7 +135,8 @@ public void handle(HttpExchange t) throws IOException {
 
   /** Main launches the server from the command line. */
   public static void main(String[] args) throws IOException, InterruptedException {
-    HttpServer jwkServer = HttpServer.create(new InetSocketAddress(8081), 0);
+    HttpServer jwkServer =
+        HttpServer.create(new InetSocketAddress(InetAddress.getLocalHost(), 8080), 0);
```

Comment:
Isn't 8080 used by the TalkerService itself?

Reply:
I found in Golang, the grpc app and shell app expose 8080, but in Java I didn't find it. In addtional, in java, we have to specify the path `/.well-known/jwks` in the public key endpoint.